src/Tools/MicroBuild/run-gitlink.ps1

```diff
@@ -0,0 +1,41 @@
+# Run GitLink on the binaries that we are producing.
```

Comment:
(if the answer was "it's no longer supported", then that should have been written in the commit message)